docs/network/interfaces_and_networks.md

```diff
@@ -22,65 +22,54 @@ Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network
 
 ## Networks
 
-Network backends are configured in `spec.networks`. A network must have
+Networks are configured in VMs `spec.template.spec.networks`. A network must have
 a unique name. Additional fields declare which logical or physical
 device the network relates to.
```

Comment:
What additional fields?
I think you can drop this odd sentence. The explanation of what each type has is described in detail later.